loki/transformations/parallel/openmp_region.py

```diff
@@ -96,14 +95,26 @@ def visit_Pragma(self, pragma, **kwargs):
         routine.body = RemoveOpenMPRegionTransformer().visit(routine.body, active=False)
 
 
-def add_openmp_regions(routine, global_variables=None, field_group_types=None):
+def add_openmp_regions(
+        routine, dimension=None, shared_variables=None, field_group_types=None
```

Comment:
```suggestion
        routine, dimension, shared_variables=None, field_group_types=None
```
I didn't realise this until the dimension was added to filter loops by loop variable, but attributes of the dimension parameter are accessed in this method, hence it shouldn't be set to ``None``.

Reply:
Nice, very good catch! Again, I'll mop this up in the rebase.